### ISSUE TITLE
Gamefaqs support 

### ISSUE THREAD

**Issue by abdellahking** [hrydgard/ppsspp#11258]
THIS IS NOT AN ISSUE
It's like a request 
Can you guys do gamefaqs support like "epsxe and soperretro 16" so we can download .txt file and put it in specific folder so we can read it on pause menu when we need it without going to anouther app ,it whold be helpful for the RPG games

Sory about the english😁

**Comment by AkiraJkr** [comment]
Let me try to rephrase that better... a feature to download the walkthrough/faqs of the game of the side to read on the pause menu...wouldn't it be better to simply open it in the browser? I don't see much of a difference...plus I feel like pausing the game while browsing would be a problem--but it's viable for mobile users because of the weird *window* system it has that keeps making the browser shut itself down. the problem is with the space, formatting and...detection of the game.

**Comment by abdellahking** [comment]
Yeah i know we simply can go to the browser,but some users (on android at least) have slow phons so going to browser and reterning to the emulator would be a nightmer 
Or simply the android will shut down  the emulator for more bettery life.

Ps:Sorry about the english

**Comment by AkiraJkr** [comment]
Yeah, that's why I referred to only Android needing this kind of feature, as I said, it is not impossible to do it, but the problem is in game name detection, how to size the faqs and show them on screen and...how much this feature would be a priority for ppsspp.

**Comment by abdellahking** [comment]
For game name detection we can use game ID like"ULES-01441.txt" for kingdom hearts:bbs 
2_for how to size them :i truetly dont know
3_and if is this request whorth the :idk i just felt that some users will need it, and if hrydgard or unknown feel this request is useless
